src/Groups/group_characters.jl

```diff
@@ -1978,8 +1978,9 @@ true
 ```
 """
 function trivial_character(tbl::GAPGroupCharacterTable)
-    val = QQAbFieldElem(1)
-    return class_function(tbl, [val for i in 1:ncols(tbl)])
+    triv = GAPWrap.TrivialCharacter(GapObj(tbl))
+    GAPWrap.SetIsIrreducibleCharacter(triv, true)
```

Comment:
Why doesn't GAP already do that? Or put another way: should GAP also be changed to do this?

Reply:
Yes, of course, this should eventually be done on the GAP side.